core/tests/test_utils.py

```diff
@@ -2904,10 +2900,10 @@ def post_json(
     # Here we use type Any because this method can return JSON response Dict
     # whose values can contain different types of values, like int, bool,
     # str and other types too.
-    def delete_json(
+    def delete_json(  # pylint: disable=dangerous-default-value
```

Comment:
Nope, we cannot use None here because testapp.delete can only accept iterable `params` and providing None here can result in backend test failuers. https://github.com/oppia/oppia/blob/c52105dd384b02dfbad49208f384a274c46534c7/core/tests/test_utils.py#L2920

Reply:
Then fix the backend tests, please.

Reply:
No no -- I mean to say `testapp.delete can only accept iterable params` and if we provide `None` here then `testapp.delete` will throw an error which results in backend test failures ( I have mentioned backend tests explicitly because this function is used only in backend tests ).

Reply:
Ah, still set it to `None` and then in the function set it to `{}` if it is `None`, I know that it might seem kinda redundant but it is safer than passing `{}` right away.

Reply:
Fixed!